.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,3 +12,6 @@
 
   Learn more about contributing: https://www.gatsbyjs.org/docs/how-to-contribute/
 -->
+
+<!-- For fixing bugs use https://github.com/gatsbyjs/gatsby/compare/?template=bugs.md -->
```

Comment:
Feel like this will be missed by a lot of people 😅

Reply:
I know... 
How can I make it more visible?

Reply:
Is it possible to have multiple pull request templates with select menu like for issues?